### DISCUSSION TITLE
This causes errors with the launcher

### DISCUSSION THREAD

**Discussion by JoeMama985** [seuyh/stellaris-dlc-unlocker#32] [Q&A]
Whenever I run it and do launcher reinstall, and open the launcher, it says "Warning, A launcher update has failed. The action failed because we encountered an internal error. Try again or restart the launcher." I 
![image](https://github.com/user-attachments/assets/a13951a1-9af6-460c-861c-7c146bfc181b)
I tried restarting the launcher which didn't work, and something to note is that only the grand archive DLC is showing up in the launcher.
This is also a problem because I paid for multiple DLCs with my own money already and they will not show up.

**Comment by seuyh** [comment]
Just ignore this
